en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -59,329 +59,192 @@ Let's see how to create a custom sequence using the WSO2 API Manager Tooling Pl
 
-6.  Navigate to the **Window** menu, click **Perspective** , **Open Perspective,** and **Other** to open the Eclipse perspective selection window.
+5.  Navigate to the **Window** menu, click **Perspective** , **Open Perspective,** and **Other** to open the Eclipse perspective selection window.
     ![](../../../assets/img/Learn/eclipse-open-perspective.png)
-7.  On the dialog box that appears, click **WSO2 API Manager** and click **OK** .
+6.  On the dialog box that appears, click **WSO2 API Manager** and click **OK** .
     ![](../../../assets/img/Learn/eclipse-open-apim.png)
-8.  On the APIM perspective, click the **Login** icon as shown below.
+7.  On the APIM perspective, click the **Login** icon as shown below.
     ![](../../../assets/img/Learn/eclipse-login.png)
-9.  On the dialog box that appears, enter the URL, username, and password of the Publisher server.
+8.  On the dialog box that appears, enter the URL, username, and password of the Publisher server.
+
     ![](../../../assets/img/Learn/eclipse-add-registry.png)
-10. On the tree view that appears, expand the folder structure of the existing API.
-11. Right-click on the `in` sequence folder and click **Create** to create a new `in` sequence.
+
+9. On the tree view that appears, expand the folder structure of the existing API.
+10. Right-click on the `in` sequence folder and click **Create** to create a new `in` sequence.
     ![](../../../assets/img/Learn/eclipse-create-in-seq.png) This is because you want the custom sequence to be invoked in the `In` direction or the request path. If you want it to be involved in the `Out` or `Fault` paths, select the respective folder under `customsequences`.
 
-        !!! tip
-    **Tip** : If you prefer not to use the registry to upload the sequence or want to engage a sequence to all APIs in WSO2 API-M at once, you can do so by saving the mediation sequence XML file in the file system. See [Adding Mediation Extensions](../../../Learn/Extensions/adding-mediation-extensions.md) for details.
+    !!! tip
+        **Tip** : If you prefer not to use the registry to upload the sequence or want to engage a sequence to all APIs in WSO2 API-M at once, you can do so by saving the mediation sequence XML file in the file system. See [Adding Mediation Extensions](../../../Learn/Extensions/adding-mediation-extensions.md) for details.
 
 
-12. Name the sequence `YahooWeatherSequence` .
-13. Your sequence now appears on the Developer Studio console. From under the **Mediators** section, drag and drop a [**Property** mediator](https://docs.wso2.com/display/EI650/Property+Mediator) to your sequence and give the following values to the property mediator.
+11. Name the sequence `pizzaOrderInSeq` .
+12. Your sequence now appears on the Developer Studio console. From under the **Mediators** section, drag and drop a [**Log** mediator](https://docs.wso2.com/display/EI650/Log+Mediator) to your sequence and give the following values.
```

Comment:
Are we referring to it as the Developer Studio console? 

From under the **Mediators** section can be reworded to, "From the **Mediators** section"

Add the values in a table